hack/generate-manifest.sh

```diff
@@ -33,6 +33,8 @@ Generate a YAML manifest for Antrea using Kustomize and print it to stdout.
         --tun (geneve|vxlan|gre|stt)  Choose encap tunnel type from geneve, gre, stt and vxlan (default is geneve)
         --verbose-log                 Generate a manifest with increased log-level (level 4) for Antrea agent and controller.
                                       This option will work only with 'dev' mode.
+        --on-delete                   Generate a manifest with antrea-agent's update strategy set to OnDelete.
+                                      This option will work only with 'kind' mode.
```

Comment:
could you add a check after command-line option parsing and print an error if `--on-delete` is used without `--kind`. Like we do for `--verbose-log` and `dev` mode.

Reply:
done, thanks